oci/private/pull.bzl

```diff
@@ -529,6 +533,7 @@ buildozer 'set digest "sha256:{digest}"' 'remove tag' 'remove platforms' {option
         build = _MULTI_PLATFORM_IMAGE_ALIAS_TMPL.format(
             name = rctx.attr.name,
             target_name = rctx.attr.target_name,
+            available_platforms = ", ".join(available_platforms),
```

Comment:
I think you should do the quote wrapping here as well, so all the "presentation" is together. 

The `'"{}"'.format` above is pretty awkward syntax too

Reply:
I am not sure I follow.

Reply:
`'"{}"'.format` is for building the buildozer command, so I wanted to use the same array

Reply:
that's why I have triple quotes in the template.

Reply:
ah okay I didn't see that the quoted version is used in another place